databricks/sdk/azure.py

```diff
@@ -15,19 +15,15 @@ class AzureEnvironment:
 ARM_DATABRICKS_RESOURCE_ID = "2ff814a6-3304-4ab8-85cb-cd0e6f879c1d"
 
 ENVIRONMENTS = dict(
-    PUBLIC=AzureEnvironment(name="AzurePublicCloud",
+    PUBLIC=AzureEnvironment(name="PUBLIC",
                             service_management_endpoint="https://management.core.windows.net/",
                             resource_manager_endpoint="https://management.azure.com/",
                             active_directory_endpoint="https://login.microsoftonline.com/"),
-    GERMAN=AzureEnvironment(name="AzureGermanCloud",
-                            service_management_endpoint="https://management.core.cloudapi.de/",
-                            resource_manager_endpoint="https://management.microsoftazure.de/",
-                            active_directory_endpoint="https://login.microsoftonline.de/"),
-    USGOVERNMENT=AzureEnvironment(name="AzureUSGovernmentCloud",
+    USGOVERNMENT=AzureEnvironment(name="USGOVERNMENT",
```

Comment:
Why do we remove the German cloud?

Reply:
This was added unnecessarily. Databricks is not run in the German cloud, and we don't include it in the [Go](https://github.com/databricks/databricks-sdk-go/blob/main/config/azure.go#L122) SDK either. It is purely in maintenance mode since 2018: https://learn.microsoft.com/en-us/azure/germany/.